.env.example

```
@@ -18,4 +18,12 @@ SPONSORSHIP_PICTURE_BUCKET=exampleKey
 WEBHOOK_SECRET=exampleKey
 NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=exampleKey
 NEXT_PUBLIC_SUPABASE_ANON_KEY=exampleKey
-NEXT_PUBLIC_SUPABASE_URL="https://bug-free-orbit-rq5vxrpqgv6357w9.github.dev/"
\ No newline at end of file
+NEXT_PUBLIC_SUPABASE_URL="https://bug-free-orbit-rq5vxrpqgv6357w9.github.dev/"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)